README.md

```diff
@@ -1,119 +1,23 @@
-  time evaluation.
-+ New Chez Scheme based backend which both compiles and runs faster than the
-  default Idris 1 backend. (Also, optionally, Racket and Gambit can be used
-  as targets).
-+ Everything works faster :).
-
-A significant change in the implementation is that there is an intermediate
-language `TTImp`, which is essentially a desugared Idris, and is cleanly
-separated from the high level language which means it is potentially usable
-as a core language for other high level syntaxes.
-
-JavaScript
-----------
-The JavaScript codegen uses the new BigInt, hence Node.js 10.4 or higher is required.
-
-Editor Plugins
---------------
-The [wiki lists the current plugins available for common text editors](
-https://github.com/idris-lang/Idris2/wiki/1-%5BLanguage%5D-Editor-support)
-and their features.
-
-Things still missing
---------------------
-
-+ Cumulativity (so we currently have Type : Type! Bear that in mind when you
-  think you've proved something :))
-+ 'rewrite' doesn't yet work on dependent types
-
-Contributions wanted
--------------------
-
-+ [Contribution guidelines](CONTRIBUTING.md)
-+ [Good first issues](https://github.com/idris-lang/Idris2/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-+ [Contributors wanted](https://github.com/idris-lang/Idris2/wiki/2-%5BCommunity%5D-Contributions-wanted)
-
-If you want to learn about Idris more, contributing to the compiler could be one
-way to do so. Just select one good first issue and ask about it on the [Discord](https://discord.gg/UX68fDs2jc) channel.
-
-Talks
------
-
-[![Edwin Brady Tells Us What's New in Idris 2 (Berlin Functional Programming Group)](https://img.youtube.com/vi/nbClauMCeds/0.jpg)](https://www.youtube.com/watch?v=nbClauMCeds "Edwin Brady Tells Us What's New in Idris 2 (Berlin Functional Programming Group)")
-
-[![Scheme Workshop Keynote (ACM SIGPLAN)](https://img.youtube.com/vi/h9YAOaBWuIk/0.jpg)](https://www.youtube.com/watch?v=h9YAOaBWuIk "Scheme Workshop Keynote (ACM SIGPLAN)")
-
-[![Idris 2 - Type-driven Development of Idris (Curry On - London 2019)](https://img.youtube.com/vi/DRq2NgeFcO0/0.jpg)](https://www.youtube.com/watch?v=DRq2NgeFcO0 "Idris 2 - Type-driven Development of Idris (Curry On - London 2019)")
-
-[![Idris 2: Type-driven Development of Idris (Code Mesh LDN 18)](https://img.youtube.com/vi/mOtKD7ml0NU/0.jpg)](https://www.youtube.com/watch?v=mOtKD7ml0NU "Idris 2: Type-driven Development of Idris (Code Mesh LDN 18)")
+If you want to learn more about Idris, contributing to the compiler could be one way to do so. Choose a [good first issue](https://github.com/idris-lang/Idris2/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) or have a look at the [contributions wanted](https://github.com/idris-lang/Idris2/wiki/2-%5BCommunity%5D-Contributions-wanted) for something more involved, then ask about it on the [Discord forum](https://discord.gg/UX68fDs2jc). See the [contribution guidelines](CONTRIBUTING.md) for more details.
```

Comment:
I think the contribution guidelines should be the first thing. They describe what we likely will and won't accept, so before looking through the issues, it's good for people to know if their work would even be likely to be merged  : )

Reply:
will see if I can reword it

Reply:
done